src/plugins/analysis/linter/code/source_code_analysis.py

```diff
@@ -33,59 +29,51 @@ class AnalysisPlugin(AnalysisBasePlugin):
     DEPENDENCIES = ['file_type']
     VERSION = '0.5'
```

Comment:
We should not forget to update the version (otherwise old analyses will not get updated)
```suggestion
    VERSION = '0.6'
```